src/vm/domainfile.h

```diff
@@ -840,8 +846,21 @@ class DomainAssembly : public DomainFile
     Volatile<bool>                          m_fHostAssemblyPublished;
     Volatile<bool>                          m_fCalculatedShouldLoadDomainNeutral;
     Volatile<bool>                          m_fShouldLoadDomainNeutral;
+    LoaderAllocator*                        m_pLoaderAllocator;
+    DomainAssembly*                         m_NextDomainAssemblyInSameALC;
```

Comment:
Shouldn't these be under ifdef FEATURE_COLLECTIBLE_ALC. This would require other related changes also to be under ifdef FEATURE_COLLECTIBLE_ALC